pkg/cmd/pr/shared/templates.go

```diff
@@ -128,54 +128,34 @@ func hasIssueTemplateSupport(httpClient *http.Client, hostname string) (bool, er
 	}
 
 	gql := graphql.NewClient(ghinstance.GraphQLEndpoint(hostname), httpClient)
-	err := gql.QueryNamed(context.Background(), "IssueTemplates_fields", &featureDetection, nil)
```

Comment:
Minor: we could consider keeping the GraphQL query name the same for backwards compatibility. That way, existing caches of this lookup would get reused. Also, nothing about this query really changed, so we don't really have much reason to change the name.

Reply:
Yup. Updated.